egs2/tedx_spanish_corpus_openslr67/asr1/local/train.txt

```diff
@@ -0,0 +1,126 @@
+TEDX_M_041
```

Comment:
Does the corpus have an official release of data split? If so, please directly follow that. If not, could you please hold the split in another repo (you may refer to `egs2/puebla_nahuatl/local/data.sh` to store the split in your own repositories)